.docker/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.4'
+
+services:
+  app:
+    image: i24-reader:app-dev
+    build:
+      context: ../
+      dockerfile: .docker/app/Dockerfile
+      target: prod
```

Comment:
why to specify the target when you only have one target in the Dockerfile ? (ie no multistage builds)